src/docs/self-hosted/index.mdx

```diff
@@ -107,6 +107,31 @@ By default Sentry sends anonymous usage statistics to the Sentry team. It helps
 
 You can disable this feature by adding `SENTRY_BEACON = False` into `sentry.conf.py` file.
 
+### Errors-only Self-hosted Sentry
+
+This feature is experimental and now in beta. Starting from 24.8.0+ onwards, users will have the ability to choose between two distinct types of self-hosted Sentry deployments, tailored to their specific needs and preferences.
+
+**Errors Only**\n
+Opt for a sleek and efficient version of self-hosted Sentry that focuses on what we do best—tackling issues head-on. This lightweight option includes these features:
```

Comment:
This paragraph is written directly addressing the user, while the one in the next section is written in a more passive style. We should pick one and stick with it.

Reply:
Also, I think this should be more technical and less markety. "Errors Only Sentry offers a small set of core features, with an eye on minimizing resource costs."